packages/typography/src/Heading.tsx

```diff
@@ -55,6 +55,11 @@ const style = css`
       ${fonts.sizes('26px', '36px')};
     }
   }
+  &[data-style='h4'] {
+    ${mq.range({ from: breakpoints.tablet })} {
+      font-family: ${fonts.serif};
+    }
```

Comment:
Stusser litt over denne. Skal vi endre font-family på større devicer?

Reply:
Hmm nei, det så litt rart ut. 

Oppdaterte nå: fjernet h4 helt fordi den matcher helt med default styling, la også til h1-resource fordi den manglet fra designet, og oppdaterte font-family for h1